test/integration/posthtml/.posthtmlrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
```

Comment:
Gives us coverage on the array branch of `loadPlugins`:

https://github.com/parcel-bundler/parcel/blob/b56b2a9f3c3ff6db2dd27a086b409a8d4af6f2bd/src/utils/loadPlugins.js#L4-L7

Other posthtmlrc is object, for complete coverage.